conf/awsbatch.config

```diff
@@ -1,10 +1,15 @@
 /*
  * -------------------------------------------------
- *  Nextflow config file for AWS Batch
+ *  Nextflow config file for running on AWS batch
  * -------------------------------------------------
- * Imported under the 'awsbatch' Nextflow profile in nextflow.config
- * Uses docker for software depedencies automagically, so not specified here.
+ * Base config needed for running with -profile awsbatch
  */
+params {
+  config_profile_name = 'AWSBATCH'
+  config_profile_description = 'AWSBATCH Cloud Profile'
+  config_profile_contact = 'Alexander Peltzer (@apeltzer)'
```

Comment:
Why is @apeltzer the contact here?

Reply:
This file was generated by the standard nf-core template 1.6. See the TEMPLATE branch [here](https://github.com/nf-core/rnafusion/blob/TEMPLATE/conf/awsbatch.config)

Reply:
Yeah, the AWS Batch profile has me because I made the one being in the template and centrally taken care of...